### DISCUSSION TITLE
Misconfigurations Detected

### DISCUSSION THREAD

**Discussion by jdog4412** [crunchy-labs/crunchy-cli#299] [Q&A] [answered]
I installed the cli however when attempting to download anything from Crunchyroll, I get the following error: `Misconfigurations detected: FFmpeg is needed to run this command`. when I use the `--ffmpeg` flag it still throws that error, not sure what I'm missing

**Comment by bytedream** [comment, marked as answer]
You need the `ffmpeg` program -> https://www.ffmpeg.org/download.html
